tools/perf/ResultsComparer/src/ResultsComparer/Bdn/BdnComparer.cs

```diff
@@ -14,49 +14,59 @@
 using ResultsComparer.Core;
 using Perfolizer.Mathematics.SignificanceTesting;
 using Perfolizer.Mathematics.Thresholds;
+using Perfolizer.Mathematics.Multimodality;
 
 namespace ResultsComparer.Bdn
 {
     /// <summary>
     /// Results comparer for BenchmarkDotNet full JSON reports.
     /// </summary>
-    internal class BdnComparer : IResultsComparer
+    public class BdnComparer : IResultsComparer
     {
-        private const string FullBdnJsonFileExtension = "full.json";
+        private const string FullBdnJsonFileExtension = ".full.json";
 
-        public Task<bool> CanReadFile(string path)
+        /// <inheritdoc/>
+        public bool CanReadFile(string path)

```

Comment:
What necessitated this change?

Reply:
Since this is CLI application, I felt like adding `async/await` everywhere was not adding any value.